### DISCUSSION TITLE
Output GELF - missing short_message key

### DISCUSSION THREAD

**Discussion by qpirsel** [fluent/fluent-bit#7372] [Q&A] [answered]
Hi!

I'm evaluating various logging systems and currently I'm testing fluentbit + graylog.

For fluentbit I'm running [version=2.1.1, commit=8144c1f4f3, pid=10312] on Windows and configured the conf file as follows:

```
[INPUT]
    Name tail
    Tag tc1-test-wp2
    Path C:\Users\john\Desktop\input.txt
    multiline.parser java

[OUTPUT]
    Name stdout
    Match *

[OUTPUT]
    Name gelf
    Match *
    Host 10.202.2.31
    Port 12201
    Mode udp
    Gelf_Short_Message_Key tc1-test-wp2
    Gelf_Host_Key svr-app20
```

But regardless what I try I will always get this error from the GELF output plugin:

```
[2023/05/12 10:45:15] [error] [output:gelf:gelf.1] error encoding to GELF
[2023/05/12 10:45:15] [error] [flb_msgpack_to_gelf] missing short_message key
```

STDOUT Output looks fine in the console. I update input.txt manually with java stack traces to test the config but I also have tried e.g. dummy as input plugin with the same error. 

Any suggestions?

**Comment by patrick-stephens** [comment, marked as answer]
I thin you're confusing tag with key - you've said to use the key `tc1-test-wp2` in the record after parsing. This key must be present.

stdout will show the tag first then the timestamp followed by key-value pairs.

**Reply by qpirsel** [reply]
Hi!

Indeed I thought the Short Message Key is a free to choose string (coincidence that it was the tag name).

I think I got it.

Thank you very much
